presto-docs/src/main/sphinx/release/release-0.180.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.180
+=============
+
+General Changes
+---------------
+
+* Fix a rare bug where rows containing only ``null`` values are not returned
+  to the client. This only occurs when an entire result page contains all
+  ``null`` values. The only known case is a query over an ORC encoded Hive table
+  that does not perform any transformation of the data.
+* Fix incorrect results when performing comparisons between values of approximate
+  data types (``REAL``, ``DOUBLE``) and columns of certain exact numeric types
+  (``INTEGER``, ``BIGINT``, ``DECIMAL``).
+* Fix memory accounting for :func:`min_by` and :func:`max_by` on complex types.
+* Fix query failure due to ``NoClassDefFoundError`` when scalar functions declared
+  in plugins are implemented with non-static methods.
+* Improve performance of map subscript from O(n) to O(1) in all cases. Previously, only maps
+  produced by certain functions and readers can take advantage of this improvement.
+* Skip unknown costs in ``EXPLAIN`` output.
+* Support :doc:`/security/internal-communication` between Presto nodes.
+* Add initial support for ``CROSS JOIN`` against ``LATERAL`` derived tables.
+* Add support for ``VARBINARY`` concatenation.
+* Add :doc:`/connector/thrift` that makes it possible to use Presto with
+  external systems without the need to implement a custom connector.
+* Add experimental ``/v1/resourceGroupState`` REST endpoint on coordinator.
+
+Hive Changes
+------------
+
+* Fix reading short decimals in the optimized Parquet reader.
```

Comment:
Skipping (used for seeks) short decimal values was broken when they are backed by a Parquet int32/int64, see #8208. 

If we want to be more specific:
`Fix skipping short decimal values in the optimized Parquet reader when they are backed by the int32/int64 types.`.